yw3487.md

```diff
@@ -0,0 +1,7 @@
+# COME4995
+
+This is a course project for Open Source Development
+
+# Ideas
+1. A software to resize laptop webpage to mobile phone ones when opening webpages on the phone. The main feature is to reorganize the webpage, so that it will be easier to read on a mobile phone. (It might be implemented through browser extensions, but Chrome doesn't support extensions on Android or ios. A Firefox add-on might do the job).
+2. A Jupyter Notebook extension to do syntax highlight for SQL code. A lot of people are using SQL plug-ins to run SQL statments in Jupyter notebook, but I didn't find anything to do some syntax highlight for the SQL code. 
```

Comment:
here are two that I know of:
https://github.com/pbugnion/jupyterlab-sql
https://blog.jupyter.org/a-jupyter-kernel-for-sqlite-9549c5dcf551

Reply:
The two links here seem to work in JupyterLab but not Jupyter notebook (though they are extremely similar) if I understand them correctly. My intended project aims to syntax highlight codes in Jupyter notebook's SQL cell magic. Something like this:
 <img width="931" alt="Screen Shot 2020-09-28 at 3 04 23 PM" src="https://user-images.githubusercontent.com/43913069/94480485-fd876680-019b-11eb-8fe0-5c84a625c229.png">
The highlighting there in the cell is the default one for IPython, which looks a bit chaotic in SQL statements. My project aims to fix this.

I hope it clarifies. Though I played with the Xeus-SQLite in the second link and it looks like a better way to work with SQL than the cell magic, which makes my project less interesting, I can start working on it anyway and change it later if other ideas come up.